.github/workflows/sycl-linux-precommit.yml

```diff
@@ -177,6 +177,14 @@ jobs:
       sycl_toolchain_archive: ${{ needs.build.outputs.artifact_archive_name }}
       sycl_toolchain_decompress_command: ${{ needs.build.outputs.artifact_decompress_command }}
 
+  # First approach to UR workflow - it's executed on any sycl changes;
+  # It downloads the UR sources from the original repo and builds it
+  # i.a. on UR hardware.
+  unified-runtime:
+    name: Unified Runtime
+    needs: [build, detect_changes]
+    if: ${{ always() && !cancelled() && contains(needs.detect_changes.outputs.filters, 'sycl') }}
+    uses: ./.github/workflows/unified-runtime.yml
```

Comment:
I think we should keep this separate from the *SYCL Pre Commit on Linux* workflow for the first iteration. When looking at the [checks](https://github.com/intel/llvm/pull/16827/checks) tab we should have a separate *Unified Rutime Pre Commit* section on the left.

Further to this, one of the suggestions made to us is to only run the UR build/test workflows when the UR source has changed. This will probably require the code to have actually moved into intel/llvm first in order to actually trigger job creation.

Reply:
make sense, updated.

as for detecting UR changes - I updated "detect_changes" workflow, and added a dummy UR changes to check trigger (it's the last commit).